src/title.ts

```diff
@@ -0,0 +1,83 @@
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+
+export interface TitleMatchConfig {
+  title?: string[];
+}
+
+export function toTitleMatchConfig(config: any): TitleMatchConfig {
+  if (!config['title']) {
+    return {};
+  }
+
+  const titleConfig = {
+    title: config['title']
+  };
+
+  if (typeof titleConfig.title === 'string') {
+    titleConfig.title = [titleConfig.title];
+  }
+
+  return titleConfig;
+}
+
+export function getTitle(): string | undefined {
+  const pullRequest = github.context.payload.pull_request;
```

Comment:
Can we instead use REST API (octokit) to get the title dynamically?

I think the `github.context` may be statically set upon workflow event trigger. This would lead to confusion about using an outdated PR title after it has changed without triggering new CI event (eg. re-running the job that uses this action after editing the PR title).